src/dto/review-and-submit-test-summary.dto.ts

```diff
@@ -0,0 +1,27 @@
+export class ReviewAndSubmitTestSummaryDTO {
```

Comment:
let's stay consistent with the code properties and spell out the word "Code"
orisCode
testTypeCode
evalStatusCode
submissionAvailabilityCode